go.mod

```diff
@@ -492,24 +548,23 @@ replace (
 	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220411220226-7b82a4e95df4
 	golang.org/x/lint => golang.org/x/lint v0.0.0-20190930215403-16217165b5de
 	golang.org/x/mod => golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
-	golang.org/x/net => golang.org/x/net v0.0.0-20220722155237-a158d28d115b
-	golang.org/x/oauth2 => golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b
+	golang.org/x/net => golang.org/x/net v0.0.0-20220921203646-d300de134e69
+	golang.org/x/oauth2 => golang.org/x/oauth2 v0.0.0-20220909003341-f21342109be1
 	golang.org/x/sync => golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
-	golang.org/x/sys => golang.org/x/sys v0.0.0-20220908164124-27713097b956
+	golang.org/x/sys => golang.org/x/sys v0.0.0-20220919091848-fb04ddd9f9c8
 	golang.org/x/term => golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	golang.org/x/text => golang.org/x/text v0.3.8
 	golang.org/x/time => golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
 	golang.org/x/tools => golang.org/x/tools v0.1.12
-	golang.org/x/xerrors => golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
-	google.golang.org/api => google.golang.org/api v0.60.0
+	golang.org/x/xerrors => golang.org/x/xerrors v0.0.0-20220609144429-65e65417b02f
+	google.golang.org/api => google.golang.org/api v0.96.0
 	google.golang.org/appengine => google.golang.org/appengine v1.6.7
-	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21
+	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20220921223823-23cae91e6737
```

Comment:
looks like this bump pulls in https://github.com/googleapis/go-genproto/pull/901 which is what triggered depending on ~every subpackage of cloud.google.com/go :-/